ci-operator/config/maistra/istio/maistra-istio-maistra-3.0.yaml

```
@@ -62,13 +74,71 @@ tests:
       env:
       - name: MAISTRA_NAMESPACE
       - name: MAISTRA_SC_POD
-      from: src
+      from: src-test
       resources:
         requests:
           cpu: 100m
           memory: 100Mi
       timeout: 2h0m0s
     workflow: servicemesh-istio-e2e-aws
+- as: maistra-istio-unit-3-0
+  commands: |
+    make -e T=-v build racetest binaries-test \
+    BUILD_WITH_CONTAINER="0" \
+    GOBIN="/gobin" \
+    GOCACHE="/tmp/cache" \
+    GOMODCACHE="/tmp/cache" \
+    XDG_CACHE_HOME="/tmp/cache" \
+    ARTIFACTS="${ARTIFACT_DIR:-}"
```

```suggestion
    make -e T=-v build racetest binaries-test \
    BUILD_WITH_CONTAINER="0" \
    GOBIN="/gobin" \
    GOCACHE="/tmp/cache" \
    GOMODCACHE="/tmp/cache" \
    XDG_CACHE_HOME="/tmp/cache" \
    ARTIFACTS="${ARTIFACT_DIR:-}" \
    2>&1 | tee "${ARTIFACT_DIR}"/make-unit.log
```

Trouble shooting timeout unit job. We can tee the make process output to a file and then check from the job artifacts tab.